docs/output.md

```diff
@@ -127,3 +127,35 @@ The resulting FASTQ files will only be present in your results directory if you
   </details>bam,bai,csi
 
   [BWA](https://bio-bwa.sourceforge.net/bwa.shtml) is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome. It consists of three algorithms: BWA-backtrack (a.k.a `bwa aln`), BWA-SW and BWA-MEM. The first algorithm is designed for Illumina sequence reads up to 100bp, while the rest two for longer sequences ranged from 70bp to 1Mbp. BWA-MEM and BWA-SW share similar features such as long-read support and split alignment, but BWA-MEM, which is the latest, is generally recommended for high-quality queries as it is faster and more accurate. BWA-MEM also has better performance than BWA-backtrack for 70-100bp Illumina reads.
+
+### Deduplication
+
+#### picard MarkDuplicates
```

Comment:
Thinking about it, do we want one section per tool, or per the name of the `results/*/` directory? I think the latter makes more sense - reduces redundency as the output files are basically the same